[docs][material-ui][joy-ui] Simplify the Quickstart section on the Usage page (#38385)
mui/material-ui: docs/data/joy/getting-started/usage/usage.md

## Quickstart

> After [installation](/joy-ui/getting-started/installation/), import any Joy UI component and start playing around with them.

Reviewer: ```suggestion
After [installation](/joy-ui/getting-started/installation/), you can import any Joy UI component and start playing around.
```
